tensorflow_datasets/core/dataset_builder.py

```diff
@@ -46,7 +60,11 @@
 from tensorflow_datasets.core.utils import gcs_utils
 from tensorflow_datasets.core.utils import read_config as read_config_lib
 
-import termcolor
+if typing.TYPE_CHECKING:  # type: ignore
```

Comment:
Why is `type: ignore` required ? This is a valid typing: https://docs.python.org/3/library/typing.html#typing.TYPE_CHECKING

Reply:
Removed